<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Saves</class>
 <widget class="QWidget" name="Window_Saves">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>138</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить изменения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>138</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Отменить изменения</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_PD">
         <property name="text">
          <string>Проектная документация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_RD">
         <property name="text">
          <string>Рабочая документация</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
